### PR TITLE
Adds Danger to Xcodeproj

### PR DESCRIPTION
**Pull request by orta**
_shoud not fail_ as I didn't make library changes which require a CHANGELOG entry.

### CODE REVIEW

**Comment by dnkoutso** [comment]
🆒

File: .travis.yml [hunk 1]
```diff
@@ -15,6 +15,15 @@ rvm:
   # TODO currently unavailable: https://github.com/travis-ci/travis-ci/issues/2918
   # - 2.0.0-p247
 
+matrix:
+  include:
+  - rvm: 2.3.1
```
Comment (segiddins) [inline .travis.yml]: maybe should use 2.4?

Comment (orta) [inline .travis.yml]: agree, fixed

**Comment by segiddins** [comment]
https://travis-ci.org/CocoaPods/Xcodeproj/jobs/219760902#L1814

**Comment by orta** [comment]
Hah, this is what I had done:

screen shot 2017-04-07 at 18 51 55

**Comment by orta** [comment]
OK, we're all good now 👍

**Pull request merged**